src/style-sheets/generic/_button.scss

```diff
@@ -0,0 +1,14 @@
+.button {
+    width: 189px;
+    height: 49px;
+    border-radius: 4px;
+    border-width: 0;
+
+    font-size: 15pt;
```

Comment:
I noticed previously you were using `em`. Is there a reason for the change now?

Reply:
Fixed it